test-cases/performance/perf-regression-latency-650gb-with-nemesis.yaml

```diff
@@ -34,6 +34,7 @@ use_hdrhistogram: true
 use_placement_group: true
 use_capacity_reservation: true
 email_subject_postfix: 'latency during operations'
+parallel_node_operations: false
```

Comment:
let put some comment here, since we are doing this to align the measurement we were doing on vnodes

we should make sure we don't make it false for tablets

Reply:
@fruch 
Why it should be "true" for tablets test? If it is "false"  in v15 branch (https://argus.scylladb.com/tests/scylla-cluster-tests/30709120-a236-4e4a-b720-65561280bf0a)

Reply:
let not change the tablets as well, so it can be comparable.